db/migrate/20150218235739_add_name_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddNameToUsers < ActiveRecord::Migration
+  def change
+  	add_column :users, :first_name, :string
+    add_column :users, :last_name, :string
```

Comment:
Consistency counts, please format your code with proper indentations. Evaluate your other code for similar issues.